src/applications/vaos/tests/e2e/vaos-helpers.js

```diff
@@ -150,6 +150,32 @@ function appointmentSubmittedTest(client) {
   return client;
 }
 
+function showMoreTest(client) {
+  client
+    .click(
+      'li[data-request-id="8a48912a6cab0202016cb4fcaa8b0038"] .additional-info-button.va-button-link',
+    )
+    .waitForElementVisible('.additional-info-content', Timeouts.slow)
```

Comment:
I think this is going to end up being a flaky test. As best I can tell, the `.additional-info-content` element will show up shortly after clicking on the link. But the service call could take longer, and will insert the message into the already existing `additional-info-content` element. I would add a pause after this line to make sure we wait for the messages call to finish.